cpp/src/strings/split/split_record.cu

```diff
@@ -79,50 +79,24 @@ std::unique_ptr<column> split_record_fn(strings_column_view const& input,
 
 enum class Dir { FORWARD, BACKWARD };
 
-/**
- * @brief Compute the number of tokens for the `idx'th` string element of `d_strings`.
- */
-struct whitespace_token_counter_fn {
-  column_device_view const d_strings;  // strings to split
-  size_type const max_tokens = std::numeric_limits<size_type>::max();
-
-  __device__ size_type operator()(size_type idx) const
-  {
-    if (d_strings.is_null(idx)) { return 0; }
-
-    auto const d_str        = d_strings.element<string_view>(idx);
-    size_type token_count   = 0;
-    auto spaces             = true;
-    auto reached_max_tokens = false;
-    for (auto ch : d_str) {
-      if (spaces != (ch <= ' ')) {
-        if (!spaces) {
-          if (token_count < max_tokens - 1) {
-            token_count++;
-          } else {
-            reached_max_tokens = true;
-            break;
-          }
-        }
-        spaces = !spaces;
-      }
-    }
-    // pandas.Series.str.split("") returns 0 tokens.
-    if (reached_max_tokens || !spaces) token_count++;
-    return token_count;
-  }
-};
-
 /**
  * @brief Identify the tokens from the `idx'th` string element of `d_strings`.
  */
 template <Dir dir>
 struct whitespace_token_reader_fn {
   column_device_view const d_strings;  // strings to split
   size_type const max_tokens{};
```

Comment:
default value of this `max_tokens` and `whitespace_token_counter_fn::max_tokens` are different.
could we make both of them same?